Makefile

```diff
@@ -137,3 +139,15 @@ debug-x86:
 
 debug-arc:
 	$(CODK_ARC_DIR)/arc32/bin/arc-elf32-gdb $(ARC_PROJ_DIR)/arc-debug.elf
+
+project:
+	@if [ -d $(PROJNAME) ]; then echo "$(PROJNAME) already exists."; exit 1; fi
+	mkdir $(PROJNAME)
+	mkdir $(PROJNAME)/arc
+	mkdir $(PROJNAME)/x86
+	mkdir $(PROJNAME)/x86/src
```

Comment:
or... just:
cp -r $(PROJ_TEMPLATE_DIR) $(PROJNAME)
if the template directory has all of the bare-bones files